src/formatHit.js

```diff
@@ -46,7 +49,7 @@ export default function formatHit({
       name: getBestHighlightedForm(hit._highlightResult.locale_names),
       city: city ? getBestHighlightedForm(hit._highlightResult.city) : undefined,
       administrative: administrative ? getBestHighlightedForm(hit._highlightResult.administrative) : undefined,
-      country: hit._highlightResult.country.value
+      country: country ? hit._highlightResult.country.value : undefined
```

Comment:
I am adding a test for that one, but for the previous one I don't know the usecase